### PR TITLE
[RFR] Use GitHub API to get a nicer demo

### PR DESCRIPTION
**Pull request by jpetitcolas**
- [x] Updating EventDrops demo code to show commits
- [x] Create a script to import commits from most starred marmelab repositories
- [x] Update demo styling

### CODE REVIEW

**Comment by jpetitcolas** [comment]
Related live coding session:
* https://www.livecoding.tv/jpetitcolas/videos/Wo4EQ-d3js-node-improving-eventdrops-demo-2
* https://www.livecoding.tv/jpetitcolas/videos/b0MXL-d3js-node-improving-eventdrops-demo-3

File: package.json [hunk 1]
```diff
@@ -61,16 +63,37 @@
...
-    { "name": "Chulki Lee" },
-    { "name": "Emmanuel Quentin" },
-    { "name": "Francois Zaninotto" },
-    { "name": "Joan Yin" },
-    { "name": "Joel Schlosser" },
-    { "name": "Jonathan Petitcolas", "email": "petitcolas.jonathan@gmail.com" },
-    { "name": "Mélodie Benmouffek" },
-    { "name": "messense" },
-    { "name": "Robin Bressan" }
+    {
```
Comment (Kmaschta) [inline package.json]: Keep the old indentation.
"It's ugly" - @jpetitcolas

**Comment by jpetitcolas** [comment]
Current demo with tooltips:

image

**Comment by jpetitcolas** [comment]
image

File: Makefile [hunk 2]
```diff
@@ -4,7 +4,7 @@ help:
 	@grep -P '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
 run: ## Run the webpack-dev-server
-	./node_modules/webpack-dev-server/bin/webpack-dev-server.js --colors --devtool cheap-module-inline-source-map --host=0.0.0.0
+	./node_modules/webpack-dev-server/bin/webpack-dev-server.js --colors --host=0.0.0.0
```
Comment (jpetitcolas) [inline Makefile]: Cheap module breaks dev server. No idea why. Will be soon fixed.

File: package.json [hunk 3]
```diff
@@ -28,11 +29,13 @@
     "extract-text-webpack-plugin": "1.0.1",
     "html-loader": "0.3.0",
     "html-webpack-plugin": "1.6.1",
+    "isomorphic-fetch": "^2.2.1",
```
Comment (jpetitcolas) [inline package.json]: Remove

**Comment by jpetitcolas** [comment]
Switching to RFR. 

Commit message is still not well formatted (using dashes instead of spaces), but let's fix it in another PR.

File: demo/bin/updateData.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    { name: 'admin-on-rest', identifier: 'marmelab/admin-on-rest' },
+    { name: 'ng-admin', identifier: 'marmelab/ng-admin' },
+    { name: 'EventDrops', identifier: 'marmelab/EventDrops' },
+    { name: 'javascript-boilerplate', identifier: 'marmelab/javascript-boilerplate' },
+    { name: 'sedy', identifier: 'marmelab/sedy' },
+    { name: 'restful.js', identifier: 'marmelab/restful.js' },
+];
+
+const setup = cb => {
+    exec(`mkdir /tmp/event-drops`, cb);
```
Comment (Kmaschta) [inline demo/bin/updateData.js]: s/mkdir/mkdir -p/

File: demo/bin/updateData.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-console */
```
Comment (Kmaschta) [inline demo/bin/updateData.js]: That's a high order script!

**Pull request merged**